tests/components/group/test_sensor.py

```diff
@@ -266,24 +266,63 @@ async def test_sensor_incorrect_state(
 
     entity_ids = config["sensor"]["entities"]
 
+    # Check that the final sensor value ignores the non numeric input
     for entity_id, value in dict(zip(entity_ids, VALUES_ERROR)).items():
         hass.states.async_set(entity_id, value)
         await hass.async_block_till_done()
 
     state = hass.states.get("sensor.test_failure")
+    assert state.state == "17.0"
+
```

Comment:
Use caplog to assert there is no logging

Reply:
Done